frontend/movie-collection/src/app/models/movie.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { Movie } from './movie';
+
+describe('Movie', () => {
+   it('should create an instance of Movie',() => {
+       expect(new Movie()).toBeTruthy();
+   });
+
+   it('should accept values', () => {
+    let movie = new Movie();
+    movie = {
```

Comment:
this test is just testing the data you have created, these asserts are not required at all